### ISSUE TITLE
validation set size

### ISSUE THREAD

**Issue by hzhz2020** [islam-nassar/semco#6]
Dear Author 

Thank you for your interesting work!
you mentioned that you tune the hyperparameter on a validation set for a single experiment (CIFAR100 - 2500 labels). I am trying to understand the experiment scenario. What is the size of the validation set, is it balanced etc?
